src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +14,30 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    public static final char CONSTW = 'w';
+
     public String[] readFromFile(String fileName) {
-        return null;
+        ArrayList<String> wordsWithWList = new ArrayList<String>();
+        String text = "";
+        try {
+            text = new String(Files.readAllBytes((Paths.get(fileName)))).toLowerCase().replaceAll(
+                    "[^A-Za-zА-Яа-я\n ]", "");
+        } catch (IOException e) {
+            throw new RuntimeException("Такого файла не существует", e);
```

Comment:
let's use english language in our programs)Anybody in the world should be able to understand that